datamade/just-spaces
# Add Group Size plugin

## Overview

Closes #139. This PR adds a custom Fobi plugin to implement the Groups PLDP question. Uses choices added to `django-pldp` [here](https://github.com/datamade/django-pldp/commit/013c2416975c3430184eecaa48c95fe65fdaef8c). 

## Testing Instructions

 * Create or edit an observational survey
 * Add a new survey question: do you see "Group Size" as an option under Observational?
* Click it. Does the help text make sense? Can you add it to your survey?
